packages/components/src/local/organisms/planning-force/index.tsx

```diff
@@ -1,14 +1,27 @@
 import cx from 'classnames'
 import L, { LatLng, latLng } from 'leaflet'
-import React, { useCallback } from 'react'
+import React, { useCallback, useState, useEffect } from 'react'
 import * as ReactDOMServer from 'react-dom/server'
-import { LayerGroup, Marker, Tooltip } from 'react-leaflet-v4'
+import { LayerGroup, Marker, useMap, Tooltip } from 'react-leaflet-v4'
 import AssetIcon from '../../asset-icon'
 import { AssetRow } from '../planning-assets/types/props'
+import { BBox, GeoJsonProperties } from 'geojson'
+import 'leaflet/dist/leaflet.css'
 import styles from './styles.module.scss'
+import useSupercluster from 'use-supercluster'
+import { PointFeature } from 'supercluster'
 import PropTypes from './types/props'
 
 export const PlanningForces: React.FC<PropTypes> = ({ assets, selectedAssets, setSelectedAssets, interactive }) => {
+  const [bounds, setBounds] = useState([
```

Comment:
sorry @lilitkarapetyan , I don't like us hard-coding a particular region in our code. We'll almost certainly not use this region for any future wargame.